src/runtime/pkg/resourcecontrol/cgroups.go

```diff
@@ -241,7 +242,7 @@ func LoadResourceController(path string) (ResourceController, error) {
 			return nil, err
 		}
 	} else if cgroups.Mode() == cgroups.Unified {
-		if IsSystemdCgroup(path) {
+		if IsSystemdCgroup(path) && sandboxCgroupOnly {
```

Comment:
What about when `cgroups.Mode() != cgroups.Unified`? Doesn't that hit the same bug?

Reply:
That's probably not a bug, `NewSandboxResourceController` has handle this case, worst case is that `LoadResourceController` will return `nil, ErrCgroupMode`